csharp-basics/exercises/Polymorphism/Persons/Animal.cs

```diff
@@ -0,0 +1,26 @@
+﻿namespace Persons
+{
+    public abstract class Animal
+    {
+        protected string _animalName;
+        protected double _animalWeight;
+        protected int _foodEaten;
+
+        protected Animal(string animalName, double animalWeight, int foodEaten)
+        {
+            _animalName = animalName;
+            _animalWeight = animalWeight;
+            _foodEaten = foodEaten;
+        }
+
+        public abstract void MakeSound();
+
+
+        public abstract void Eat(Food food, int count);
+       
+        public virtual string ToString()
```

Comment:
nebus override? :? te man šķiet vajadzētu arī būt warning par to ka tāda metode jau ir un šī to esošo slēpj?